data-model/src/main/avro/Event.avdl

```diff
@@ -56,5 +56,7 @@ protocol EventProtocol {
     union { null, org.hypertrace.core.datamodel.eventfields.http.Http } http = null;
     union { null, org.hypertrace.core.datamodel.eventfields.grpc.Grpc } grpc = null;
     union { null, org.hypertrace.core.datamodel.eventfields.sql.Sql } sql = null;
+
+    union { null, string } service_name = null;
```

Comment:
Have we decided to go this route?

Reply:
@tim-mwangi This is what this ticket - https://traceableai.atlassian.net/browse/ENG-4585 mentioned. But, let a @buchi-busireddy comment.

Reply:
Yes.

Reply:
We already have service name populated inside `jaegerFields` on the Event. Do we also need it outside?

Reply:
Correct. @anujgoyal1 this field will replace the `service_name` from `jaegerFields` so we should deprecate that in avro compatible way while adding the new field.